diagnostics/ENSO_RWS/LEVEL_01/get_clima_in.py

```diff
@@ -0,0 +1,39 @@
+import numpy as np

```

Comment:
@jhafner2 get_clima_in and get_data_in are similar, and can probably be one routine. You could also move get_dims.py to the same module (not every subroutine needs to be in a separate module).